networkx/algorithms/bipartite/matching.py

```diff
@@ -269,27 +269,6 @@ def eppstein_matching(G, top_nodes=None):
 
         # did we finish layering without finding any alternating paths?
         if not unmatched:
-            unlayered = {}
```

Comment:
It does indeed appear the `unlayered` dict is unused, but I'm leery about deleting the `TODO` and associated commentary without a little more background. @jfinkels can you provide any insight?

Reply:
Yeah, let's just comment out `unlayered` code bits for now, rather than deleting it for now. @Mjh9122 could you just comment out all of the code bits instead of removing them? Thanks!!